.travis.yml

```diff
@@ -1,28 +1,54 @@
-dist: trusty
+dist: xenial
 sudo: false
 language: rust
-cache:
-  directories:
-    - $HOME/.cargo
-    - $HOME/.cache
-    - $TRAVIS_BUILD_DIR/target
-before_cache:
-  - find $TRAVIS_BUILD_DIR/target/debug -maxdepth 1 -type f -delete
+cache: cargo
```

Comment:
Does it include `$HOME/.cache`?

Reply:
Because the Windows platform was only supported at last year, I did not find useful information about the cache in Windows platform.
For insurance, just cache the cargo directory.
After testing, the cache seems to have little effect on speed, but I can restore these deleted items and leave them unchanged if necessary.